packages/ERTP/src/amountMath.chainmail

```diff
@@ -25,98 +26,92 @@ struct Extent {
+  getMathHelpersName -> (String);
 
-  /**  Make units from an extent by adding the brand. */
-  make(allegedExtent :Extent) -> (Units);
+  /**  Make an amount from an extent by adding the brand. */
+  make(allegedExtent :Extent) -> (Amount);
 
   /**
-   * Make sure these units (or extent) are valid and return them if so.
+   * Make sure this amount (or extent) is valid and return it if so.
    */
-  coerce(allegedUnitsOrExtent :Units) -> (Units);
+  coerce(allegedAmountOrExtent :Amount) -> (Amount);
 
   /** Remove the brand and return the extent. */
-  extent(units :Units) -> (Extent);
+  extent(amount :Amount) -> (Extent);
 
-  /** Return the units representing an empty amount. This is the
-   * identity element in ExtentOps.
+  /** Return the amount representing an empty amount. This is the
+   * identity element in MathHelpers.
    */
-  empty() -> (Units);
+  getEmpty() -> (Amount);
 
-  /** Return true if the Units is empty. Otherwise false. */
-  isEmpty(units :Units) -> (boolean);
+  /** Return true if the Amount is empty. Otherwise false. */
+  isEmpty(amount :Amount) -> (boolean);
 
-  /** Returns true if the leftUnits contains the rightUnits. */
-  includes(leftUnits :Units, rightUnits :Units) -> (boolean);
+  /** Returns true if the leftAmount is greater than or equal to the rightAmount. */
+  isGTE(leftAmount :Amount, rightAmount :Amount) -> (boolean);
 
   /** 
-   * Returns true if the leftUnits equals the rightUnits. We assume
-   * that if includes is true in both directions, equals is also true 
+   * Returns true if the leftAmount equals the rightAmount. We assume
+   * that if isGTE is true in both directions, isEqual is also true 
    */
-  equals(leftUnits :Units, rightUnits :Units) -> (boolean);
+  isEqual(leftAmount :Amount, rightAmount :Amount) -> (boolean);
 
   /**
-   * Returns a new units that includes both leftUnits and rightUnits.
+   * Returns a new amount that is the union of both leftAmount and rightAmount.
    *
-   * For fungible units this means adding the extents. For other kinds of
-   * units, it usually means including both.
+   * For fungible amount this means adding the extents. For other kinds of
+   * amount, it usually means including all of the elements from both
+   * left and right. 
    */
-  with(leftUnits :Units, rightUnits :Units) -> (Units);
+  add(leftAmount :Amount, rightAmount :Amount) -> (Amount);
 
   /**
-   * Returns a new units that includes the portion of leftUnits not included
-   * in rightUnits. If leftUnits doesn't include rightUnits, throw an error.
-   */
-  without(leftUnits :Units, rightUnits :Units) -> (Units);
+  * Returns a new amount that is the leftAmount minus the rightAmount
+  * (i.e. everything in the leftAmount that is not in the
+  * rightAmount). If leftAmount doesn't include rightAmount
+  * (subtraction results in a negative), throw  an error. Note: 
+  * because we throw an error if the result is "negative", this is NOT
```

Comment:
Gets the point across. But more correct to say "because the left amount must include the right amount, this is NOT equivalent to set subtraction." The reason the text as written is technically not correct is that set subtraction does not result in a negative when the condition is violated.

I'm still uncomfortable with the name `subtract` for this reason, but we can leave it as `subtract` for this PR.

Reply:
That sounds good